lib/licensee/content_helper.rb

```diff
@@ -158,6 +171,13 @@ def strip_all_rights_reserved(string)
       strip(string, ALL_RIGHTS_RESERVED_REGEX)
     end
 
+    def strip_cc0_optional(string)
+      return string unless string.include? 'cc0'
+      string = strip(string, CC_LEGAL_CODE_REGEX)
```

Comment:
If this is an alternate title, I believe if you add a reference in https://github.com/benbalter/licensee/blob/1553efdd953953818272a60b3ac2185c8aa0e4c4/lib/licensee/license.rb#L88-L92, it should be stripped by existing normalizers.

For context, here's the existing cc0 title regex:

```
/(?i-mx:creative\ commons\ zero\,?\ (?:version\ |v)?1\.0\ universal)|cc0\-1\.0/
```

Reply:
It's really not. Titles are used by the reference matcher (for README files) and `CREATIVE COMMONS LEGAL CODE` doesn't make any sense there. It's some cruft added to the license text by the license steward, for better or worse, that is optional/often isn't deployed in the wild, but often is.